tests/evaluations/task_manager/test_correctness.py

```
@@ -0,0 +1,299 @@
+
+<reference_outputs>
+{reference_outputs}
+</reference_outputs>
+"""
+
+
+def create_async_graph_caller_for_task_manager(graph):
+    """
+    Create a task manager graph caller that properly handles the dataset format.
+    """
+    logger = get_logger("task_manager_graph_caller")
```

Ruff (0.11.9)

Local variable `logger` is assigned to but never used

Remove assignment to unused variable `logger`

(F841)